src/reducers/articlesReducer.js

```diff
@@ -0,0 +1,22 @@
+import { FETCH_CATEGORY_SUCCESS, FETCH_CATEGORY_FAILURE, FETCH_CATEGORY_REQUEST } from '../actions/actionTypes';
+
+const articlesReducer = (state = {}, action) => {
+  const { type, categoryName, payload } = action;
+  switch (type) {
+    case FETCH_CATEGORY_REQUEST:
+      return ({
+        ...state,
+        [categoryName]: {}
+      });
+    case FETCH_CATEGORY_SUCCESS:
+    case FETCH_CATEGORY_FAILURE:
+      return ({
+        ...state,
+        [categoryName]: payload
+      });
+    default:
+      return state;
```

Comment:
If you set the initial state to {}. Wouldn't all the default values in the store be cleared out?

Reply:
No, this "state" is only scoped to the `articleReducer`

Reply:
Noted